app/src/main/java/org/dhis2/Bindings/Extensions.kt

```diff
@@ -62,6 +62,7 @@ fun AppCompatActivity.app(): App {
 }
 
 val Int.dp: Int
-    get() = (this / Resources.getSystem().displayMetrics.density).toInt()
+    get() = (this * Resources.getSystem().displayMetrics.density).toInt()
+
 val Int.px: Int
-    get() = (this * Resources.getSystem().displayMetrics.density).toInt()
\ No newline at end of file
+    get() = (this / Resources.getSystem().displayMetrics.density).toInt()
```

Comment:
Nice to have. I'd make a bit more clear.

Int.FromPixelToDP / Int.FromDpToPixel

Reply:
No, no. I changed it so that if you do 23.dp thats means 23 dps. And if you do 23.px is 23 pixels. Which I think it is way better and clearer than fromTo methods